firstvoices/backend/search/queries/text_matching.py

```diff
@@ -0,0 +1,52 @@
+from elasticsearch_dsl import Q
+
+BASE_BOOST = 1.0  # neutral default
+
```

Comment:
There's probably some opportunities to reuse these in other search endpoints, but I left it for another time.